beacon/beacon_setup_service.cpp

```diff
@@ -530,5 +530,12 @@ bool BeaconSetupService::CheckQualComplaints()
   return true;
 }
 
+bool BeaconSetupService::InQual(Identifier index) {
```

Comment:
What type is Identifier here? It's being copied into the function, I also am not sure of what the reason is for constructing set<Identifier> from the set that comes back from beacon->qual(). BaseDkg already has an InQual method?

Reply:
It's the validator index so `uint32` which is why I didn't bother passing it by reference. Thanks, I didn't realise BaseDkg has a InQual method. I will change it so that it uses that.